### PR TITLE
Replace apache.commons Base64 with jdk Base64

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

`java.util.Base64` is more performant than `org.apache.commons.codec.binary.Base64`.
We already use `java.util.Base64` in some of the Unit tests, but we still use `org.apache.commons.codec.binary.Base64` in the code.
Replace the apache commons Base64 with the ones from the jdk.
**Tested scenarios**

All tests are passing

### CODE REVIEW

File: templates/libraries/okhttp-gson/JSON.mustache [hunk 1]
```diff
@@ -218,7 +217,7 @@ public class JSON {
                     return null;
                 default:
                     String bytesAsBase64 = in.nextString();
-                    return Base64.decodeBase64(bytesAsBase64);
+                    return Base64.getDecoder().decode(bytesAsBase64);
```
Comment (jillingk) [inline templates/libraries/okhttp-gson/JSON.mustache:220]:
```suggestion
                    return Base64.getDecoder().decode(bytesAsBase64);
```
This is actually not the proper mustache template dir. You need the other one :)

Comment (jillingk) [reply templates/libraries/okhttp-gson/JSON.mustache:220]: -> jersey3

Comment (AlexandrosMor) [reply templates/libraries/okhttp-gson/JSON.mustache:220]: There is not base64 related code in `templates/libraries/jersey3/`, right ? @jillingk

File: src/main/java/com/adyen/util/HMACValidator.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 import com.adyen.model.notification.Amount;
 import com.adyen.model.notification.NotificationRequestItem;
-import org.apache.commons.codec.binary.Base64;
+import java.util.Base64;
 import org.apache.commons.codec.binary.Hex;
```
[the remark this answers is not in the document]
Comment (AlexandrosMor) [reply src/main/java/com/adyen/util/HMACValidator.java:26]: @Aleffio I think this outside of this ticket scope, right ?

**Review by Aleffio — APPROVED** [review]

**Review by jillingk — APPROVED** [review]

**Pull request merged**